README.md

```diff
@@ -37,18 +32,13 @@ The environment variables are:
 In order to mock your environment so you can **test** things, you can have a
 script called `env.sh` for example with the contents of:
 ```
-export OPENDAP_DIR='/tmp/opendap'
-export PUBLIC_DIR='/tmp/public'
 export ARCHIVE_DIR='/tmp/archive'
 export INCOMING_DIR='/tmp/incoming'
-export OPENDAP_IMOS_DIR="$OPENDAP_DIR/1/IMOS/opendap"
-export PUBLIC_IMOS_DIR="$PUBLIC_DIR"
-export ARCHIVE_IMOS_DIR="$ARCHIVE_DIR"
 export WIP_DIR='/tmp/wip'
 export DATA_SERVICES_DIR="$PWD"
 export LOG_DIR='/tmp/log'
 
-mkdir -p $OPENDAP_IMOS_DIR $PUBLIC_IMOS_DIR $ARCHIVE_IMOS_DIR $WIP_DIR $LOG_DIR
+mkdir -p $ARCHIVE_DIR $WIP_DIR $LOG_DIR
```

Comment:
I think it is redundant, as the VM takes care of this. In general, most of this env section in the README is redundant. But @lbesnard wanted it.

The po box should take care of all of this AFAIK.